src/main/java/com/example/codeE/model/course/Course.java

```diff
@@ -1,28 +1,65 @@
 package com.example.codeE.model.course;
 
-import java.util.Date;
-
+import com.example.codeE.model.topic.Topic;
+import com.example.codeE.util.DateTimeUtil;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import lombok.NonNull;
 import lombok.Setter;
 
+import java.time.LocalDateTime;
+import java.util.List;
+
 @Getter
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
+@Entity(name = "course")
 public class Course {
-    @NonNull
+    @Id
+    @NotBlank(message = "Course id is required")
+    @Column(name = "course_id")
     private String courseId;
-    @NonNull
+
+    @Column(name = "course_name")
+    @NotBlank(message = "Course name is required")
     private String courseName;
-    @NonNull
+
+    @Column(name = "main_teacher")
+    @NotBlank(message = "Teacher ID is required")
     private String mainTeacher;
+
+    @Column(name = "semester")
+    @NotBlank(message = "Semester is required")
     private String semester;
-    @NonNull
-    private Date dateCreate;
+
+    @Column(name = "created_date")
+    @NotBlank(message = "Created date is required")
+    private String createdDate;
+
+    @Column(name = "updated_date")
+    @NotBlank(message = "Updated date is required")
```

Comment:
Do we need this? In Database, the value is default to `NOW()`, so we do not need to set this value.